### ISSUE TITLE
do you train new Tecogan model?

### ISSUE THREAD

**Issue by FrankLinxzx** [km2ii/JP109A_TG-PLUS#33]
Do you train new Tecogan model?or just use the already version model ,Thx

**Comment by km2ii** [comment]
In the update, isn't it?
SE-4X — 20200922
SE-2X — 20200922
SE-4X-R1 — 20210809
SE-4X-S1 — 20210830
UE-4X — 20210830

**Comment by FrankLinxzx** [comment]
> In the update, isn't it?
> SE-4X — 20200922
> SE-2X — 20200922
> SE-4X-R1 — 20210809
> SE-4X-S1 — 20210830
> UE-4X — 20210830

i tried to train tecogan model , batch size =4 but my loss  just up and down  0.3x to 0.4x    global steps about 400000 , should i try to fix learning rate or something else?

**Comment by km2ii** [comment]
1.Your global loss is too big, the official version loss is very small. 10,000 steps around 0.09-0.12.
2.400,000 steps are enough for testing, manually modify config.ini to add your model, do not use the keywords UE,SE, and capitalize the model name, otherwise it may report an error.
3.If the test result is not satisfactory, you should give up to continue training. Check the dataset or something else.
4.About around 950,000 steps, training is close to balance and there is little point in continuing. 
5.Reducing the learning rate has little effect and requires longer training, the official version seems to be optimal.

**Comment by FrankLinxzx** [comment]
> 1.Your global loss is too big, the official version loss is very small. 10,000 steps around 0.09-0.12.
> 2.400,000 steps are enough for testing, manually modify config.ini to add your model, do not use the keywords UE,SE, and capitalize the model name, otherwise it may report an error.
> 3.If the test result is not satisfactory, you should give up to continue training. Check the dataset or something else.
> 4.About around 950,000 steps, training is close to balance and there is little point in continuing.
> 5.Reducing the learning rate has little effect and requires longer training, the official version seems to be optimal.

i can test use javplayer ,  but not sure  how much learning rate should be in  the special global steps  
(now i use learning rate= 5e-05  for global_step 67880  All_loss_Gen 0.4x

**Comment by km2ii** [comment]
no, it's global step 10000. epoch about 1-2.

**Comment by km2ii** [comment]
default,no need to change.

**Comment by FrankLinxzx** [comment]
> default,no need to change.

ok      thx
